kvmagent/kvmagent/plugins/ovn.py

```
@@ -407,6 +454,95 @@ def ovn_del_port(self, req):
 
         return jsonobject.dumps(rsp)
 
+    @kvmagent.replyerror
+    def ovn_check_local_port(self, req):
+        cmd = jsonobject.loads(req[http.REQUEST_BODY])
```

Ruff (0.14.0)

Local variable `cmd` is assigned to but never used

Remove assignment to unused variable `cmd`

(F841)